mysql-test/suite/galera/t/galera_lock_table.test

```diff
@@ -66,3 +66,78 @@ show create table wsrep_cluster;
 #
 use mysql;
 select count(*) from wsrep_cluster;
+
+#
+# Test scenario:
+# 1) unload wsrep_provider
+# 2) lock tables/instance for backup
+# 3) start transaction
+# 4) execute statement that causes WSREP transaction rollback, but does not
+#    involve any storage engine
+# 5) check that backup lock is still present
+#
+
+--let $no_backup_locks_txt = No backup locks expected
+--let $backup_locks_text = One backup lock expected
+
+--connection node_2
+--let $wsrep_provider_orig = `SELECT @@wsrep_provider`
+--let $wsrep_cluster_address_orig = `SELECT @@wsrep_cluster_address`
+
+SET GLOBAL wsrep_provider='none';
+
+
+--let $assert_text = $no_backup_locks_txt
+--let $assert_cond = [SELECT COUNT(*) FROM performance_schema.metadata_locks WHERE OBJECT_TYPE='BACKUP TABLES'] = 0
```

Comment:
In all relevant places, please source `assert.inc`.

Reply:
What a silly mistake!. Fixed